issue CMake error

`CMake Warning at cmake/SearchForStuff.cmake:203 (message):
  ENABLE_TESTS was on but gtest was not found, unit tests will not be enabled
Call Stack (most recent call first):
  CMakeLists.txt:30 (include)


CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
X11_XCB_LIBRARIES
    linked by target "GSdx" in directory /home/park/Downloads/pcsx2/plugins/GSdx

-- Configuring incomplete, errors occurred!
See also "/home/park/Downloads/pcsx2/build/CMakeFiles/CMakeOutput.log".
See also "/home/park/Downloads/pcsx2/build/CMakeFiles/CMakeError.log".
make: *** No targets specified and no makefile found.  Stop.
make: *** No rule to make target 'install'.  Stop.

`

--- a/debian-packager/control
+++ b/debian-packager/control
@@ -20,6 +20,7 @@ Build-Depends: cmake (>= 2.8.5),
     libwxgtk3.0-dev,
     libxml2-dev,
     libx11-dev,
+    libx11-xcb-dev,
     locales | locales-all,
     portaudio19-dev,
     zlib1g-dev

